app/code/Magento/Sales/view/adminhtml/layout/sales_order_create_load_block_search_grid.xml

```diff
@@ -8,7 +8,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="content">
-            <block class="Magento\Sales\Block\Adminhtml\Order\Create\Search\Grid" name="serach.grid" as="grid"/>
+            <block class="Magento\Sales\Block\Adminhtml\Order\Create\Search\Grid" name="search.grid" as="grid"/>
```

Comment:
Hi @manuelcanepa. Thank you for your contribution. It's obvious that there's a typo in the block name. But we may not simply change the block name and introduce a backward compatibility issue in that way. There might be 3rd-party modules that reference this block by its current name. By making this change we will break the mentioned modules